SECURITY.md

```diff
@@ -0,0 +1,3 @@
+## Reporting a Vulnerability
+
+If you discover any security related issues, please email YOUREMAILHERE instead of using the issue tracker.
```

Comment:
Use `security@monicahq.com`
```suggestion
If you discover any security related issues, please email security@monicahq.com instead of using the issue tracker.
```